pytx/pytx/common.py

```diff
@@ -241,7 +244,8 @@ def objects(cls,
                 owner=owner,
                 status=status,
                 review_status=review_status,
-                share_level=share_level
+                share_level=share_level,
+                sort_order=sort_order
```

Comment:
Can you please add a trailing comma? That will let us keep the blame correct. This PR doesn't really modify the share_level line above, for example.